services/server/routers/comments.py

```diff
@@ -0,0 +1,40 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+from pydantic import BaseModel
+from typing import Optional
+
+from services.server.db.crud import get_db
+from starlette.requests import Request
+from sqlalchemy.orm import Session
+from authlib.integrations.starlette_client import OAuth, OAuthError
+
+from services.server.db import crud
+
+from services.server.routers.oauth import starlette_config
+from starlette.responses import JSONResponse
+
+from services.server.schemas import Comment
+
+comment = APIRouter(
+   tags=["comment"]
+)
+oauth = OAuth(starlette_config)
+
+
+@comment.post('/api/comments')
```

Comment:
Please [refer to this example in relations branch](https://github.com/NBNARADHYA/prep-project-4.1.2/blob/5818e5e58da9f4cd8c6b0ffa91a7e3c48f7b7ff7/services/server/main.py#L73) on how to make use of authentication middleware in a protected route. You will directly get the user as a function parameter.